src/angularComponent/ngcOmniboxController.js

```diff
@@ -63,6 +63,10 @@ export default class NgcOmniboxController {
     }
   }
 
+  $onInit() {
+    this._requireMatch = this.requireMatch === 'true';
```

Comment:
Why don't we just use `<` binding? I'm not sure I like this private property. We already default to `false`, so `<`'s default behavior matches ours.